content/blog/2019-11-17-salut-bucharest.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Salut Bucharest
+author: sathya
+type: post
+date: 2019-11-17T04:51:13+00:00
+url: /2019/11/17/salut-bucharest/
+categories:
+  - Life
+tags:
+  - bucharest
+  - romania
+  - career
+  - immigration
+
+---
+
+
+It's been a while since I posted anything, so lets get this started again. As I had tweeted couple of weeks back, I have moved to Bucharest, Romania. 
+
+<!--more-->
+
+
+{{< tweet 1182251512334077952 >}}
+
+{{< tweet 1184008163546284032 >}}
+
+So some of the obvious questions would be:
+
+- Why Romainia?
+- How did it happen?
+
+Let's take on them one by one:
+
+### Why Romania?
+
+To which I say why not? I've been looking at moving out for a while now, and the US was never an option, especially with the current visa situation(and Jo not being able to work with a dependent visa). Europe is what Jo and me had been looking at since a while. I've been working with my colleagues from Bucharest, Romania since past couple of couple of years and this gave me a chance to bring up a internal transfer request within my team. With the approvals in place, we were finally able to start the process. However, as with immigration & inter-country transfers, this was just the start of a long road of lots of paperwork and documentation shuttle.
```

Comment:
You might want to add a space between `situation` and, `(and Jo`. Also, the next sentence has `couple of ` twice.